test/distributed/org/apache/cassandra/distributed/test/ring/CleanupDuringRangeMovementTest.java

```diff
@@ -51,11 +51,11 @@ public class CleanupDuringRangeMovementTest extends TestBaseImpl
     public void cleanupDuringDecommissionTest() throws Throwable
     {
         ExecutorService executor = Executors.newSingleThreadExecutor();
-        try (Cluster cluster = builder().withNodes(2)
-                                        .withTokenSupplier(evenlyDistributedTokens(2))
-                                        .withNodeIdTopology(NetworkTopology.singleDcNetworkTopology(2, "dc0", "rack0"))
-                                        .withConfig(config -> config.with(NETWORK, GOSSIP))
-                                        .start())
+        try (Cluster cluster = init(builder().withNodes(2)
```

Comment:
I missed the existence of the `init` method, its usage seems to simplify the code a bit although I had the impression the existence of `createKeyspaceWithTable` added a bit more clarity when reading the test case from top to bottom.